cineast-core/src/test/java/org/vitrivr/cineast/core/db/dao/MetadataTest.java

```diff
@@ -96,9 +96,6 @@ void setupTest() {
   @AfterEach
   void tearDownTest() {
     dropTables();
-
-    /* Close all readers / writers. */
-    CineastIOUtils.closeQuietly(this.objWriter, this.segWriter, this.objReader, this.segReader);
```

Comment:
This should probably be moved to the @AfterAll method

Reply:
Did that.